--- a/src/corelib/global/minimum-linux_p.h	2019-04-09 11:51:26.000000000 +0200
+++ b/src/corelib/global/minimum-linux_p.h	2019-05-22 21:58:35.957666329 +0200
@@ -78,27 +78,9 @@
  * - statx                      4.11                    QT_CONFIG(statx)
  */
 
-#if QT_CONFIG(statx) && !QT_CONFIG(glibc)
-// if using glibc, the statx() function in sysdeps/unix/sysv/linux/statx.c
-// falls back to stat() for us.
-// (Using QT_CONFIG(glibc) instead of __GLIBC__ because the macros aren't
-// defined in assembler mode)
-#  define MINLINUX_MAJOR        4
-#  define MINLINUX_MINOR        11
-#  define MINLINUX_PATCH        0
-#elif QT_CONFIG(getentropy)
-#  define MINLINUX_MAJOR        3
-#  define MINLINUX_MINOR        17
-#  define MINLINUX_PATCH        0
-#elif QT_CONFIG(renameat2)
-#  define MINLINUX_MAJOR        3
-#  define MINLINUX_MINOR        16
-#  define MINLINUX_PATCH        0
-#else
 #  define MINLINUX_MAJOR        2
 #  define MINLINUX_MINOR        6
 #  define MINLINUX_PATCH        28
-#endif
 
 #define MINIMUM_LINUX_VERSION  QT_VERSION_CHECK(MINLINUX_MAJOR, MINLINUX_MINOR, MINLINUX_PATCH)
 
